src/main/java/com/couchbase/lite/store/ForestDBViewStore.java

```diff
@@ -227,11 +228,9 @@ public Status updateIndexes(List<ViewStore> inputViews) throws CouchbaseLiteExce
             mapBlocks.add(map);
 
             String docType = delegate.getDocumentType();
-            if (docType != null) {
-                if (docTypes == null)
-                    docTypes = new ArrayList<String>();
-                docTypes.add(docType);
-            }
+            docTypes.add(docType);
```

Comment:
It seems there is possibility that docType is null (from following code).
Is adding null in docTypes OK?